hyperspyui/widgets/extendedqwidgets.py

```diff
@@ -129,8 +128,8 @@ class ExRememberPrompt(ExMessageBox):
 
     def __init__(self, *args, **kwargs):
         super(ExRememberPrompt, self).__init__(*args, **kwargs)
-        cb = QtWidgets.QCheckBox(tr("Remember this choice"))
-        self.setCheckBox(cb)
+        self.cb = QtWidgets.QCheckBox(tr("Remember this choice"))
```

Comment:
For good measure, could you also add `self` as the second argument here? It will set the message box as the parent of the checkbox. This might have been sufficient on its own, but having both is probably good.

Reply:
Sure, done!